bin/fin

```diff
@@ -4545,6 +4546,19 @@ run_cli ()
 	fi
 }
 
+# Reload Cron Configuration
+_cron_reload ()
```

Comment:
There is no need to put underscore in the beginning of the function name without a reason. Underscore does not make "private" functions. We put it to functions which names are ambiguous because command line utilities with that name already exist or it is a reserved word e.g. `mysql`, `bash`, `exec`